.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # The codeowners for the Octopus repository are
-* @JiscSD/Octopus-Dev
+* @Jisc/Octopus-Dev
```

Comment:
this might be a breaker..

Reply:
this will never make any sense really, we either remove it or use @Jisc (which wont work in jisc-SDG), the side effect is PRs wont automatically get assigned to the team, or we update it once merged and remove it from Jisc/octopus (that will give a conflict though and the repos wont be exact copies)